modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCart.testcase

```diff
@@ -346,6 +346,106 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-7870. CanChangeQuantityOfASubscriptionProductInMiniCart"
+	@priority = "4"
+	test CanChangeQuantityOfASubscriptionProductInMiniCart {
+	property portal.acceptance = "false";
+
+		task ("Given a Minium site with an account") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			ApplicationsMenu.gotoSite(site = "Minium");
+
+			CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Account Management");
+
+			CommerceAcceleratorsInitializer.createNewAccountViaFront(accountName = "Commerce Account");
+		}
+		
+		task ("And given a product with subscription") {
+			CommerceNavigator.gotoPortlet(
+				category = "Product Management",
+				portlet = "Products");
+
+			CommerceProducts.searchProduct(search = "ABS Sensor");
+
+			CommerceNavigator.gotoEntry(entryName = "ABS Sensor");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Subscription");
+
+			Click(locator1 = "CommerceEntry#DELIVERY_SUBSCRIPTION_TOGGLE");
+
+			Button.clickPublish();
+
+			Alert.viewSuccessMessage();
+		}
+		
+		task ("When the product with description is added to the cart") {
```

Comment:
the description is not relevant for the test, we can omit that

Reply:
yes, it should be subscription, right, thanks!